tests/yam/agama/import_agama_profile.pm

```diff
@@ -7,7 +7,11 @@
 use base Yam::Agama::patch_agama_base;
 use strict;
 use warnings;
-use testapi qw(assert_script_run data_url get_required_var select_console script_run);
+use testapi qw(
+  assert_script_run data_url get_var
+  get_required_var select_console
+  script_run background_script_run
```

Comment:
script_run is not needed, please add one per line, in alphabetical order, it is more clear to read.

Reply:
Changed.